app/models/insurance_plan.rb

```diff
@@ -0,0 +1,3 @@
+class InsurancePlan < ApplicationRecord
+  belongs_to :car
```

Comment:
Mmm, esto no esta bien del todo. Un auto tiene muchos seguros eso es posible. Y un seguro tiene muchos autos. La relaciones es muchos a muchos. 
1. Usar una relacion [has_and_belongs_to_many](https://guides.rubyonrails.org/association_basics.html#the-has-and-belongs-to-many-association)
2. Falta la relacion del proveedor con el plan de seguro no?